integration_tests/models/schema_tests/data_test.sql

```diff
@@ -1,11 +1,11 @@
 select
     1 as idx,
     '2020-10-21' as date_col,
-    cast(0 as{{ dbt_utils.type_float() }}) as col_numeric_a,
-    cast(1 as{{ dbt_utils.type_float() }}) as col_numeric_b,
+    cast(0 as {{ dbt_utils.type_float() }}) as col_numeric_a,
```

Comment:
Good catch, not sure how that happened!

Reply:
Interestingly, the old version compiles to 
```sql
cast(0 as
    float64
) as col_numeric_a
```
so shouldn't throw an error but still a good change.

Reply:
for me it was compiling as `cast(0 asfloat64) as col_numneric_a` so it was throwing an error